src/pages/namespace/Gateway/Consumers/Table/Row/SecretInput.tsx

```diff
@@ -0,0 +1,32 @@
+import { Eye, EyeOff } from "lucide-react";
```

Comment:
This could maybe be renamed to "PasswordInput" or "HiddenInput" -- my thought is: If we would later reuse it as a general component, it might be confused with our record type "secret".

It's even used for a "password" right now, so sticking with that term makes sense.

Reply:
Good one,  I will change that!